CHANGELOG

```diff
@@ -1,4 +1,26 @@
 # Changelog
+## [0.2.1] - 2019-01-13
+### Breaking Changes
+ - Deprecation in `libnmstate.schema`:
+    - 'LinuxBridge.GROUP_FORWARD_MASK' -> 'LinuxBridge.Options.GROUP_FORWARD_MASK'
+    - 'LinuxBridge.MAC_AGEING_TIME' -> 'LinuxBridge.Options.MAC_AGEING_TIME'
+    - 'LinuxBridge.MULTICAST_SNOOPING' -> 'LinuxBridge.Options.MULTICAST_SNOOPING'
+
+### New Features
+ - Support SR-IOV VFs configuration
+ - Introduce libnmstate.__version__
+
+### Bug Fixes
+ - Fix verification of route rules with 2+ route tables
+ - Support reapply on IPv6 changes
+ - Fix disable IP stack after enabled DHCP
+ - Fix MAC address pattern to support bigger addresses
+ - Report missing OVS NM plugin on profile activation
+ - Ignore existing unmanaged and unknown type bridge ports
+ - Fix bridge type interface state parsing
+ - Assume removed interface as success
```

Comment:
This needs a little bit more context or more of an overview, for example: `Fix success check after interface deletion`

Reply:
Done! Thanks!